backend/capellacollab/projects/capellamodels/modelsources/git/routes.py

```diff
@@ -142,3 +148,16 @@ def update_git_model_by_id(
     )
 
     return updated_git_model
+
+
+@router.delete(
+    "/{git_model_id}",
+    dependencies=[
+        Depends(ProjectRoleVerification(required_role=ProjectUserRole.MANAGER))
+    ],
+)
+def delete_git_model_by_id(
+    db_git_model: DatabaseGitModel = Depends(get_existing_git_model),
+    db: Session = Depends(get_db),
+) -> DatabaseGitModel:
+    crud.delete_git_model(db, db_git_model)
```

Comment:
```suggestion
@router.delete(
    "/{git_model_id}",
    status_code=204,
    dependencies=[
        Depends(ProjectRoleVerification(required_role=ProjectUserRole.MANAGER))
    ],
)
def delete_git_model_by_id(
    db_git_model: DatabaseGitModel = Depends(get_existing_git_model),
    db: Session = Depends(get_db),
):
    crud.delete_git_model(db, db_git_model)
```
Removing the return type and adding the status code 204 will no longer return `null`.